ansi/buffer_test.go

```diff
@@ -29,3 +29,26 @@ func Benchmark_PrintableRuneWidth(b *testing.B) {
 		}
 	})
 }
+
+func Test_Truncate(t *testing.T) {
```

Comment:
More test cases in table will be helpful 😄

Reply:
Totally.